zerver/lib/event_schema.py

```diff
@@ -1790,6 +1790,15 @@ def check_update_message(
 check_update_message_flags_remove = make_checker(update_message_flags_remove_event)
 
 
+group_setting_type = UnionType(
+    [
+        int,
+        DictType(
+            required_keys=[("direct_members", ListType(int)), ("direct_subgroups", ListType(int))]
+        ),
```

Comment:
I would probably line-wrap this so each field gets its own line; you can tell `ruff-format` that's what you want by adding trailing commas.

Reply:
Updated.